specification/cognitiveservices/OpenAI.Inference/models/translation.create.tsp

```diff
@@ -0,0 +1,45 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+import "./transcription.create.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Translation request.")
+model TranslationOptions {
+
+    @doc("The audio file to translate. Binary data.")
```

Comment:
More of a docs pass thing, but I suspect we won't need to (and likely shouldn't) describe type as part of the comment. Actual library exposure will be "idiomatic and friendly" (like file types) and overdocumenting the wire detail could promote confusion.